daemon/container_operations_unix.go

```diff
@@ -354,13 +354,18 @@ func killProcessDirectly(container *container.Container) error {
 	// In case there were some exceptions(e.g., state of zombie and D)
 	if process.Alive(pid) {
 		// Since we can not kill a zombie pid, add zombie check here
-		isZombie, err := process.Zombie(pid)
+		status, err := process.Status(pid)
 		if err != nil {
 			logrus.WithError(err).WithField("container", container.ID).Warn("Container state is invalid")
 			return err
```

Comment:
Not new; we should probably wrap this error in a `errdefs.System` as well (and perhaps include the container ID in this message).

Not for this PR though; probably better to handle separately.